libcontainer/system/sysconfig.go

```diff
@@ -10,3 +10,7 @@ import "C"
 func GetClockTicks() int {
 	return int(C.sysconf(C._SC_CLK_TCK))
 }
+
+func GetLongBit() int {
+	return int(C.sysconf(C._SC_LONG_BIT))
```

Comment:
FYI, `_SC_LONG_BIT` doesn't exist in musl, so Alpine's going to need a workaround/patch here

cc @ncopa